balancer_wrapper.go

```diff
@@ -450,13 +450,14 @@ func (acbw *acBalancerWrapper) healthListenerRegFn() func(context.Context, func(
 	if acbw.ccb.cc.dopts.disableHealthCheck {
 		return noOpRegisterHealthListenerFn
 	}
+	cfg := acbw.ac.cc.healthCheckConfig()
+	if cfg == nil {
+		return noOpRegisterHealthListenerFn
+	}
 	regHealthLisFn := internal.RegisterClientHealthCheckListener
 	if regHealthLisFn == nil {
 		// The health package is not imported.
-		return noOpRegisterHealthListenerFn
-	}
-	cfg := acbw.ac.cc.healthCheckConfig()
-	if cfg == nil {
+		channelz.Error(logger, acbw.ac.channelz, "Health check is requested but health package is not imported.")
```

Comment:
This would be very helpful, thank!

I mentioned this on my issue #8590 since I missed the magic import: Could we consider making the magic import unnecessary? I assume we need this indirection to reduce dependencies of the core gRPC library or something like that.

Reply:
I just realized that I had this review comment as a draft, but had forgotten to hit enter.

I now remember that @arjan-bal mentioned that one possibility for explicitly asking users to blank import the health package is to avoid taking a proto dependency. Let's wait on @dfawley to decide on this.

Reply:
Yes, the reason for this was to avoid the health proto dependency.  We already depend on proto, and have done so since before 1.0, because the proto codec is included by default (as is the status proto).

How big is the health proto dependency itself?